java/core/pom.xml

```diff
@@ -128,20 +128,6 @@
             <version>${dependency.google-auth-library-oauth2-http.version}</version>
         </dependency>
 
-
-        <!-- RFC 2822 validator -->
-        <!-- see https://github.com/bbottema/email-rfc2822-validator -->
-        <dependency>
-            <groupId>com.github.bbottema</groupId>
-            <artifactId>emailaddress-rfc2822</artifactId>
-            <version>2.3.1</version>
-        </dependency>
-        <dependency> <!-- for email validation to work in lib above; used in RESTApiSanitizerImpl, PseudonymizerImpl as of June 2023 -->
-            <groupId>com.sun.mail</groupId>
-            <artifactId>jakarta.mail</artifactId> <!-- CDDL + GPLv2 with classpath exception license, so ideally would remove -->
-            <version>[1.6.3,1.9.9)</version>
```

Comment:
this was optional dep of their email address parser, which was tricky to manage - as we also have it in worklytics code base - so some conflicts between them.

additionally - the CDDL + GPLv2 stuff was possibly a problem - certainly a more exotic license type to contend with; and really all they used out of here was `InternetAddress` pojo (? or does it validate domains???) - which wasn't adding value for our use cases.  We don't care about email addresses being *valid* in the sense of sending mail to them ... just syntactically/pragmatically valid.